server/README.md

```diff
@@ -108,3 +109,47 @@ This demo implementation rejects login attempts by the demo user.
 ### UserHasLoggedIn
 
 This demo implementation logs a message to the demo channel whenever a user logs in.
+
+# Plugin Settings
+
+The following settings are available in the demo plugin system console page to demonstrate what is available via the [Mattermost Plugin Settings Schema](https://developers.mattermost.com/extend/plugins/manifest-reference/#settings_schema).
+
+![system-console-demo-settings](docs/demo-settings.png)
+
+## Channel Name
+
+A `text` setting type to define the channel that the demo plugin will create and use to post messages.
+
+## Username
+
+A `text` setting type to define the username value that will be given to the demo user. This is set via 
```

Comment:
`This is set via` accidentally left at the end of the line

Reply:
Thanks for catching that